Unit Tests/SMLHelper/SelfCheckingDictionary.cs

```diff
@@ -4,7 +4,7 @@
 namespace QModManager.Tests.SMLHelper
 {
     [TestFixture]
-    internal class SelfCheckingDictionaryTests
+    internal class SelfCheckingDictionary
```

Comment:
This is a `TestFixture` with unit tests that validate the functionality of the `SelfCheckingDictionary` class.  
"`SelfCheckingDictionaryTests`" is an appropriate name by convention.

Renaming this test class to the same name as the class its testing just looks confusing.